metricbeat/mb/testing/modules.go

```diff
@@ -79,6 +81,7 @@ func (m *TestModule) Config() mb.ModuleConfig                   { return m.ModCo
 func (m *TestModule) UnpackConfig(to interface{}) error         { return m.RawConfig.Unpack(to) }
 func (m *TestModule) UpdateStatus(_ status.Status, _ string)    {}
 func (m *TestModule) SetStatusReporter(_ status.StatusReporter) {}
+func (m *TestModule) Logger() *logp.Logger                      { return nil }
```

Comment:
I believe this needs to be deleted too due to https://github.com/elastic/beats/pull/43973/commits/35b8f9d351de2ab3a338caf76ac7761ac3ee1a12